src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader reader = Files.newBufferedReader(Paths.get(fromFileName))) {
+            String line;
+
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                switch (parts[0]) {
+                    case "supply":
+                        supply += Integer.parseInt(parts[1]);
```

Comment:
```suggestion
                String[] parts = line.split(DELIMITER);
                switch (parts[OPERATION_INDEX]) {
                    case "supply":
                        supply += Integer.parseInt(parts[VALUE_INDEX]);
```
please create constants